File: Programmers_64065/solution.js
function solution(s) {
  var answer = [];
  let sArray = s.split("},{");
  sArray[0] = sArray[0].substring(2);
  sArray[sArray.length - 1] = sArray[sArray.length - 1].substring(
    0,
    sArray[sArray.length - 1].length - 2
  );
  const sArray2 = [];

  sArray.map((arr) => {
    sArray2.push(arr.split(","));
  });
  for (let i = 0; i < sArray2.length; i++) {
    let tmp = 0;
    sArray2.map((row) => {
      if (row.length === 1) {
        answer.push(row[0] * 1);
        tmp = row[0];
      }
    });
    sArray2.map((row) => {
      row.map((findTmp, index) => {
        if (findTmp === tmp) {
          row.splice(index, 1);
        }
      });
    });
  }
  return answer;
}

solution("{{123}}");

File: Programmers_64065/solution.py
def solution(s):
    answer = []
    s = s[2:len(s)-2].split("},{") # 앞 뒤 중괄호 제거 및 ","으로 분리
    data = [list(map(int, item.split(","))) for item in s] # str형 s를 list로 변경
    data.sort(key = lambda x: len(x)) # list의 길이 오름차순으로 정렬
    
    answer.append(data[0][0])
    for idx in range(1, len(data)) :
        answer.append(sum(data[idx]) - sum(data[idx-1]))

    return answer